src/test/java/com/ericsson/ei/flowtests/FlowTestBase.java

```diff
@@ -203,8 +227,14 @@ JsonNode getJSONFromFile(String filePath) throws IOException {
     }
 
     // count documents that were processed
-    private long countProcessedEvents(String database, String collection) {
-        MongoDatabase db = getFlowTestConfigs().getMongoClient().getDatabase(database);
+    private long countProcessedEvents(String database, String collection) {    	
+    	MongoClient mongoClient = null;
+    	if (systemTest) {
```

Comment:
Every where else there is if (!sy...) so the flow test code is at the above part of the if statement. Try to keep the same in the file so it is easier to follow the 2 different tests.

Reply:
Done.